### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by julz0815**
a better visualisation for customer to use this example, removing unnecessary parts and adding 'runs-on: ubunut-latest'

### CODE REVIEW

**Review by sinat101 — CHANGES_REQUESTED** [review]
i think a compile/build step is important to demonstrate a real world use case of someone uploading their compiled binary to the product. most people host their source code on a repo like github and then build and compile it before uploading the binary to be scanned. i suggest adding the following after the check out of the main branch:

`      - name: Build with Maven # Compiling the .war binary from the checked out repo source code to upload to the scanner in the next step
          run: mvn -B package --file app/pom.xml`

File: README.md [hunk 1]
```diff
@@ -106,21 +106,30 @@ The following example will upload all files contained within the folder_to_uploa
...
+    
+jobs:
+  static_analysis:
+    name: Static Analysis
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Check out main branch
+        uses: actions/checkout@v2
+
```
Comment (sinat101) [inline README.md:120]: i think a compile/build step is important to demonstrate a real world use case of someone uploading their compiled binary to the product. most people host their source code on a repo like github and then build and compile it before uploading the binary to be scanned. i suggest adding the following after the check out of the main branch:

`      - name: Build with Maven # Compiling the .war binary from the checked out repo source code to upload to the scanner in the next step`
`          run: mvn -B package --file app/pom.xml`

**Review by sinat101 — CHANGES_REQUESTED** [review]
#14

File: README.md (hunk 1 above)
Comment (tjarrettveracode) [reply README.md:120]: Agree that this would be a good addition -- I will merge @julz0815 's changes and then make an update.

**Comment by tjarrettveracode** [comment]
Thanks both for your input! This looks much better.

**Pull request merged**